python/coinbase-agentkit/coinbase_agentkit/wallet_providers/eth_account_wallet_provider.py

```diff
@@ -1,49 +1,56 @@
 from decimal import Decimal
 from typing import Any
 
-from eth_account import Account
 from eth_account.datastructures import SignedTransaction
 from eth_account.messages import encode_defunct
+from eth_account.signers.local import LocalAccount
 from pydantic import BaseModel
 from web3 import Web3
 from web3.middleware import SignAndSendRawMiddlewareBuilder
 from web3.types import BlockIdentifier, ChecksumAddress, HexStr, TxParams
 
-from ..network import Network
+from ..network import NETWORK_ID_TO_CHAIN, NETWORK_ID_TO_CHAIN_ID, Network
 from .evm_wallet_provider import EvmWalletProvider
 
 
 class EthAccountWalletProviderConfig(BaseModel):
     """Configuration for EthAccountWalletProvider."""
 
-    private_key: str
-    rpc_url: str
-    chain_id: int
+    account: LocalAccount
+    network_id: str
+
+    class Config:
+        """Configuration for EthAccountWalletProvider."""
+
+        arbitrary_types_allowed = True
```

Comment:
Had to add this to silence a Pydantic error due to the addition of `LocalAccount` here